ReactAndroid/processor/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
A java module won't need this.

Reply:
@gengjiawen Thank you. Removed the unused AndroidManifest.xml.